rasa/core/agent.py

```diff
@@ -132,7 +132,6 @@ def _load_and_set_updated_model(
 
     async with model_server.session() as session:
         try:
-            set_log_level()
```

Comment:
we still need to set it somewhere else though

Reply:
Yes, any idea where though?
Also I will try to look into why these warnings are printed. They don't seem to be warning us of anything, or are they?

Reply:
mhm usually it should only be set in functions called by the CLI

Reply:
The root cause if this is actually the function `enable_async_loop_debugging` in `utils.io`. It sets the `slow_callback` parameter to 1ms, causing asyncio to raise a warning everytime a task takes longer than 1ms.
I think it would be better to address the issue there, by raising that value, to something like 100ms (in some cases).

Reply:
yes that should be set to something more reasonable

Reply:
but it is two different things: `set_log_level` is generally about the logging level and the `slow_callback` is just for the asyncio logging (what I am trying to say: we should fix the asyncio warnings but the `set_log_level` call still needs to go somewhere)

Reply:
@tabergma I think you looked into this for a while - do you know why this logging configuration call is in here?

Reply:
From the original PR I got the impression that the call to `set_log_level` was there to hide the warnings.

Reply:
As we open a new session we need to set the log level again. Otherwise, the default values are taken and it differs from the logging behaviour the user set.

Reply:
What does 'session' mean here?

Reply:
`aiohttp.ClientSession`